src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,110 @@
+<!---{
+  "preview": "default",
+  "default": "preview",
+  "draft": true,
+  "experiment": true,
+  "component": "amp-form"
+  }--->
+<!--
+#### Introduction
+
+This sample showcases how to build a comment section in AMP HTML.
```

Comment:
If I provide guidance, does it mean I need to implement it in this sample? Or should i link to https://github.com/ampproject/amp-publisher-sample?